todo_app/data/mongo_db_items.py

```diff
@@ -0,0 +1,78 @@
+from todo_app.repository.mongo_db_repository import MongoDbRepository
+from todo_app.models.item import Item
+from todo_app.models.status import Status
+
+
+class MongoDbItemProvider:
+    def __init__(self):
+        self.mongo_db_repository = MongoDbRepository()
+
+    def get_items_by_status(self):
+        """
+        Fetches all saved items from MongoDb, partitioned by status.
+
+        Returns:
+            not_started: The list of items with Not Started status
+            in_progress: The list of items with In Progress status
+            done: The list of items with Done status
+        """
+        not_started = self.map_mongo_db_items_to_display_class(
```

Comment:
Given this map function is only used in this class, would it be nicer to just let it know about the repository itself, so you only need to pass the status through? I.e. having it as a `get_items_as_display_class` or something